docs/content/sips/005-manifest-redesign.md

```diff
@@ -61,31 +85,47 @@ In order to decouple triggers and components, we move triggers to new top-level
 ```toml
 [[trigger.http]]
 route = "/world"
```

Comment:
super nit: can we use a different example here than "world" for the route and handler component? At first I got confused by the lines below about whether this referred to a wit world. Using "hello-world" instead might make it even more explicit that this just an arbitrary string.